actionpack/test/controller/params_wrapper_test.rb

```diff
@@ -34,7 +34,9 @@ def self.attribute_names
     end
   end
 
-  class Person
+  class Person < ActiveRecord::Base
```

Comment:
I don't believe `ActiveRecord` is available in Action Pack tests, you'll have to manually list out the methods you want 😬